README.md

```diff
@@ -45,7 +48,26 @@ pip install morphology-workflows
 
 ## Usage
 
-This workflow is based on the ``luigi`` library but can be run via the command line interface:
+### Create inputs
+
+The inputs should consist in:
+
+* a directory containing the input morphologies.
+* a CSV file with the following columns:
+    1. ``index``: the row index.
+    2. ``morph_path``: the path to the morphology file.
+    3. ``morph_name``: the name of the morphology.
+    4. ``brain_region``: the brain region in which the morphology is located.
+* a ``luigi.cfg`` file containing the configuration for all the tasks.
+* a ``logging.conf`` file containing the logging configuration.
+
+The [examples](https://github.com/BlueBrain/morphology-workflows/tree/main/examples) folder contains
+examples for the ``luigi.cfg`` and ``logging.conf`` files.
+
+### Run the workflows
+
+This workflow is based on the ``luigi`` library but can be run via the command line interface. For
+example, you can run the ``Curate`` workflow with the following command:
 
 ```bash
 morphology_workflows --local-scheduler Curate
```

Comment:
now i wonder if we cannot hide the --local-scheduler as default option in our cli?

Reply:
Indeed, we could use local scheduler by default, it would be simpler. I will try to add it.